drivers/timer/nrf_rtc_timer.c

```diff
@@ -40,6 +40,13 @@ BUILD_ASSERT(CHAN_COUNT <= RTC_CH_COUNT, "Not enough compare channels");
 #define ANCHOR_RANGE_END (7 * COUNTER_SPAN / 8)
 #define TARGET_TIME_INVALID (UINT64_MAX)
 
+#define NRF_RTC_INT_ALL_MASK (RTC_INTENSET_TICK_Msk |		\
+			      NRF_RTC_INT_OVERFLOW_MASK |	\
```

Comment:
Use the same type of named variables here, replace the _MASK one with a _Msk one

Reply:
same as above